karma.mc.config.js

```diff
@@ -51,10 +51,36 @@ module.exports = function(config) {
       // This will make karma fail if coverage reporting is less than the minimums here
       thresholds: !isTDD && {
         global: {
+          statements: 93,
```

Comment:
I think this is ok. Our average went down but it shouldn't affect our test quality since adding more code to `.jsm` files which had 100% coverage will decrease this average and will catch it.
We can slowly increase these numbers as we add more tests to .jsx files.